src/pages/forgot-password/forgot-password.html

```diff
@@ -1,11 +1,11 @@
 <ion-content padding class="mainConatiner">
   <div class="container _flex-box">
     <div class="_flex-box _justify-content-center imageConatiner">
-      <img src="assets/imgs/image1.jpg" class="imageAdj">
+      <img src="assets/imgs/image1.jpg" class="imageAdj" />
```

Comment:
unwanted change

Reply:
img is self closing tag so its coming here by prettier.